### PR TITLE
Improved required field validation for SAML idp details

### LINKED ISSUE #1946: [UX Review] SAML IdP details & SAML IdP add:  Improve the validation when the required fields are not filled
### Describe the bug

Add hint words under the field. Reference the images below.

### Version

15.0.2

### Expected behavior

Show "Required" under the field

image

### Actual behavior

image

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by dominikkawka**
close #1946 

image

### CODE REVIEW

File: apps/admin-ui/src/identity-providers/add/DescriptorSettings.tsx [hunk 1]
```diff
@@ -103,9 +103,7 @@ const Fields = ({ readOnly }: DescriptorSettingsProps) => {
         fieldId="kc-sso-service-url"
         isRequired
         validated={
-          errors.config?.authorizationUrl
```
Comment (jonkoops) [inline apps/admin-ui/src/identity-providers/add/DescriptorSettings.tsx:106]: Should this not be `singleSignOnServiceUrl` instead?

**Comment by jonkoops** [comment]
@dominikkawka could you make the title of the PR a bit more descriptive and without the issue number? Preferably mention the issue with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) in the description so it is linked correctly.

File: apps/admin-ui/src/identity-providers/add/DescriptorSettings.tsx (hunk 1 above)
Comment (dominikkawka) [reply apps/admin-ui/src/identity-providers/add/DescriptorSettings.tsx:106]: yes it should be, I've pushed another commit just now changing it to `singleSignOnServiceUrl`

**Review by jonkoops — APPROVED** [review]
LGTM!

**Review by agagancarczyk — APPROVED** [review]

**Pull request merged**